meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/apalis-imx8/lmp.cfg

```diff
@@ -39,16 +40,22 @@ CONFIG_ENV_IS_IN_FAT=y
 CONFIG_ENV_FAT_INTERFACE="mmc"
 CONFIG_ENV_FAT_DEVICE_AND_PART="0:1"
 CONFIG_USE_BOOTCOMMAND=y
-CONFIG_BOOTCOMMAND="fatload mmc 0:1 ${scriptaddr} /boot.itb; setenv verify 1; source ${scriptaddr}"
+CONFIG_BOOTCOMMAND="setenv verify 1; source ${scriptaddr}"
```

Comment:
as now both iMX8QM-based targets support pre-loaded boot.itb from U-Boot FIT, also makes sense to drop it completely from `IMAGE_BOOT_FILES_sota_mx8qm` in "lmp-machine-custom.inc":
```
IMAGE_BOOT_FILES_sota_mx8qm = "imx-boot u-boot.itb boot.itb"
```

Reply:
I'll add this change on top of the current series